website/content/docs/job-specification/sidecar_service.mdx

```diff
@@ -57,6 +57,8 @@ job "countdash" {
 - `tags` <code>(array&lt;string&gt;: nil)</code> - Custom Consul service tags
   for the sidecar service.
 
+- `meta` <code>(map&lt;string|string&gt;: nil)</code> - Specifies arbitrary KV metadata pairs.
```

Comment:
Can we move `meta` up above `port` so they're in alphabetical order?

Reply:
> Regarding `Diff` updates, I think there is no need to apply any updates for this PR, since the only field added is the meta field, which is a `map[string]string`. Anyway, I have added a test case for the scenario you mentioned.
Also, I tested this implementation as you commented, and it seems to be working as expected.

Ah, looks like the diff for this field passes thru the `flatmap.Flatten`, which I didn't realize actually works for maps of primitives (ref [`flatmap.go#L55`](https://github.com/hashicorp/nomad/blob/main/helper/flatmap/flatmap.go#L55)). But thanks for testing!